decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -79,5 +79,10 @@ def extended_navigation_bar(items, max_items: 5)
         max_items: max_items
       }
     end
+
+    def organization_colors
+      css = current_organization.colors.each.map { |k, v| "--#{k}: #{v};" }.join
```

Comment:
Shouldn't we filter those values that are blank?

Reply:
Is a jsonb object, so there will be no empty values. We could validate in the model before save in case a forced changed value in the apparence form. Is ok to you?

Reply:
Ok!